### PR TITLE
Add pytorch notebook

### PR DESCRIPTION
**Pull request by nnegrey**
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/cloudml-samples/381)

### CODE REVIEW

File: notebooks/pytorch/training.ipynb [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ---------------------------------------\n",
+    "# Read in the data\n",
```
Comment (gogasca) [inline notebooks/pytorch/training.ipynb]: Nit: can we move the comments in Code Cell to a Markup Cell. Same below

File: notebooks/pytorch/training.ipynb [hunk 2]
```diff
@@ -0,0 +1,306 @@
+{
```
Comment (gogasca) [inline notebooks/pytorch/training.ipynb]: nit: Change filename to: Training with PyTorch.ipynb ?

File: notebooks/pytorch/training.ipynb [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+    "    test_loss /= len(test_loader.dataset)\n",
+    "    print('\\nTest set:\\n\\tAverage loss: {:.4f}'.format(test_loss))\n",
+    "    print('\\tAccuracy: {}/{} ({:.0f}%)\\n'.format(\n",
+    "            correct,\n",
+    "            (len(test_loader) * test_loader.batch_size),\n",
+    "            100. * correct / (len(test_loader) * test_loader.batch_size)))"
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (gogasca) [inline notebooks/pytorch/training.ipynb]: Question: Is there an easy way to add local predictions?

File: notebooks/pytorch/training.ipynb (hunk 1 above)
Comment (nnegrey) [reply notebooks/pytorch/training.ipynb]: Done

File: notebooks/pytorch/training.ipynb (hunk 2 above)
Comment (nnegrey) [reply notebooks/pytorch/training.ipynb]: Done

File: notebooks/pytorch/training.ipynb (hunk 3 above)
Comment (nnegrey) [reply notebooks/pytorch/training.ipynb]: Like `gcloud` or just like a simple prediction?

It runs 10 batches and outputs something like:
```
[10,     6] loss: 0.678
[10,    12] loss: 0.694
[10,    18] loss: 0.712
[10,    24] loss: 0.692
[10,    30] loss: 0.659
[10,    36] loss: 0.679
[10,    42] loss: 0.685
[10,    48] loss: 0.700

Test set:
	Average loss: 3.2214
	Accuracy: 111/208 (53%)
```

Comment (gogasca) [reply notebooks/pytorch/training.ipynb]: simple prediction no gcloud. (Native PyTorch)

**Comment by gogasca** [comment]
LGTM

File: notebooks/pytorch/training.ipynb (hunk 3 above)
Comment (nnegrey) [reply notebooks/pytorch/training.ipynb]: Added

**Pull request merged**